spec.bs

```diff
@@ -666,8 +773,9 @@ To <dfn>handle a redeem response</dfn>, given [=request=] |request| and [=respon
 
 1. If |request|'s [=request/header list=] [=header list/contains|does not contain=] <a http-header>Sec-Private-State-Token</a>, return null.
 1. If |response|'s [=response/header list=] [=header list/contains|does not contain=] <a http-header>Sec-Private-State-Token</a>, return a [=network error=].
-1. Let |header| be the result of [=header list/get|getting=] <a http-header>Sec-Private-State-Token</a> from |response|'s [=response/header list=].
-1. If |header| is empty, return null.
+1. Let |rawHeader| be the result of [=header list/get|getting=] <a http-header>Sec-Private-State-Token</a> from |response|'s [=response/header list=].
```

Comment:
This should use https://fetch.spec.whatwg.org/#concept-header-list-get-structured-header, right? And then check that it's a https://httpwg.org/specs/rfc8941.html#binary?

Reply:
Added a step for the base64 decoding, unfortunately as currently implemented, this is just a base64 in a header and not the full structured header format.

Reply:
@aykutbulut Since you were looking at some of the structured header bits?

Reply:
PST spec is incompatible with the [RFC 8941](https://www.rfc-editor.org/rfc/rfc8941.html#name-byte-sequences) in its current form. Created #268 for this.